.github/workflows/ui-rm_head_upgrade.yaml

```diff
@@ -33,8 +33,19 @@ on:
         default: v1.28.8+k3s1
         type: string
         required: true
+      k8s_version_upgrade_ds_cluster:
+        description: Proceed with K8s version Upgrade (True/False)
```

Comment:
Given that this is upgrade test and default value is set to `false`, please consider small comment to explain why or remind not to forget to set to `true`

Reply:
Yes, I will add it and push it.